src/commands/synthetics/run-test.ts

```diff
@@ -205,7 +205,11 @@ export class RunTestCommand extends Command {
 
     // Override with file config variables
     const configPath = this.configPath ?? this.config.configPath
-    this.config = await parseConfigFile(this.config, configPath)
+    try {
+      this.config = await parseConfigFile(this.config, configPath)
+    } catch (error) {
+      this.reporter!.log(error.message)
+    }
```

Comment:
It is missing a new line at end of the error message.
Could you also make it work like current behavior does: if no `configPath` has been passed, we try to load the default config file path but do not throw an error if we do not find it, if a `configPath` has been passed, we throw an error if it is not found.
In both cases, we throw an error if it fails to be parsed.